Source/IC.cpp

```diff
@@ -26,4 +55,136 @@ TaylorGreen::TaylorGreen()
     }
 }
 
+ViscosityTest::ViscosityTest()
+{
+    amrex::ParmParse pp(identifier());
+    pp.query("density", m_op.density);
+
+    amrex::Vector<amrex::Real> velocity{AMREX_D_DECL(0, 0, 0)};
+    pp.queryarr("velocity", velocity, 0, AMREX_SPACEDIM);
+    for (int n = 0; n < velocity.size(); n++) {
+        m_op.velocity[n] = velocity[n];
+    }
+
+    pp.query("waveLength", m_op.waveLength); // ns:wavelength of the
```

Comment:
instead of comments, prefer meaningful variable names. e.g. here you can remove the comment and leave as is or be more specific and use `perturbation_wavelength`